Makefile

```diff
@@ -64,16 +64,14 @@ test:
 
 .PHONY: all build update_deps release clean check_deps check test
 
-bin/cucumber: Gemfile
-	bundle install --binstubs=bin --path=vendor/bundle
-
 # Our fake api server generates its own self-signed certificate, and
 # outputs some noice on standard error; redirecting it to /dev/null
 # cleans up the output a little bit.
 #
 # Depends on the target/release/delivery executable having been built
-cucumber: release bin/cucumber
-	bin/cucumber 2>/dev/null && rm -rf features/tmp
+cucumber: release Gemfile
+	chef exec bundle install
```

Comment:
I don't think we need this `bundle install` thing, we are not even using it when we run cucumber. If we wanted to use it we would have to do `chef exec bundle exec cucumber` and then you will see everything blowing up. So I am +1 on removing the `Gemfile` and use cucumber from ChefDK.